daemon/src/engine/qcommon/common.cpp

```diff
@@ -1318,7 +1320,7 @@ void Com_WriteConfigToFile( const char *filename, void (*writeConfig)( fileHandl
 		return;
 	}
 
-	FS_Printf( f, "// generated by Unvanquished, do not modify\n" );
+	FS_Printf( f, "// generated by " PRODUCT_NAME ", do not modify\n" );
```

Comment:
Would make more sense to have this be the game name rather than the engine. (People were already wtf is daemon when we released Unvanquished.)